axon-framework/events/event-processors/README.md

```diff
@@ -253,12 +308,213 @@ public interface ErrorHandler {
+                                                                    .maxSequenceSize(256)
+                                                                    .build()));
+    }
+}
+```
+{% endtab %}
+
+{% tab title="Spring Boot AutoConfiguration" %}
+```java
+@Configuration
+public class DeadLetterQueueExampleConfig {
+
+    public static final String PROCESSING_GROUP = "deadLetterProcessor";
+
+    @Autowired
+    public ConfigurerModule configure() {
+        return configurer ->
+                configurer.eventProcessing(eventProcessingConfigurer -> eventProcessingConfigurer.registerDeadLetterQueue(
+                        PROCESSING_GROUP,
+                        configuration -> JpaSequencedDeadLetterQueue.builder()
+                                                                    .processingGroup(
+                                                                            PROCESSING_GROUP)
+                                                                    .transactionManager(configuration.getComponent(
+                                                                            TransactionManager.class))
+                                                                    .entityManagerProvider(
+                                                                            configuration.getComponent(
+                                                                                    EntityManagerProvider.class))
+                                                                    .serializer(
+                                                                            configuration.serializer())
+                                                                    .maxSequences(256)
+                                                                    .maxSequenceSize(256)
+                                                                    .build()));
+    }
+}
+```
+{% endtab %}
+
+{% endtabs %}
+You can set the maximum amount of sequences that are saved (defaults to 128) and the maximum amount of dead-letters in a
+sequence (also defaults to 128). If these thresholds are exceeded an exception will be thrown and the event processor
+will stop processing.
+The sequence id is determined by the sequencing policy. By default, the sequence identifier is the aggregate id.
+
+After fixing the issue the events can be handled again by using the `process` function, in this case it will process the
+first matching event of type ErrorEvent:
+
+{% tabs %}
+{% tab title="Process first dead-letter in the queue of type ErrorEvent " %}
+
+```java
+public class DeadletterProcessor {
+
+    private final EventProcessingConfiguration eventProcessingConfiguration;
+
+    public DeadletterProcessor(EventProcessingConfiguration eventProcessingConfiguration) {
+        this.eventProcessingConfiguration = eventProcessingConfiguration;
+    }
+
+    public void repairErrorEvent() {
```

Comment:
Technically speaking, there's no guarantee it'll be repaired. That depends on whether the handler process really is fixed.
Hence I think it's better to call this method `retryErrorEventSequence`.